src/test/java/com/karankumar/bookproject/backend/entity/AuthorTest.java

```diff
@@ -114,4 +125,50 @@ void notRemoveNonOrphans() {
         // then
         assertThat(authorService.count()).isOne();
     }
+    
+    @Test
+    void notAcceptNullFirstName() {
+    	// given
+    	Author authorWithNullFirstName = new Author(null, "Gaarder");
+    	
+    	Set<ConstraintViolation<Author>> violations = validator.validateProperty(authorWithNullFirstName, "firstName");
```

Comment:
As this exceeds our 100 character column limit, could you drop the initialisation part down one line?